schemas/public/onpremises-kfd-v1alpha2.json

```
@@ -1784,6 +1784,20 @@
           "type": "array",
           "description": "The additional static clients for dex"
         },
+        "expiry": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "signingKeys" : {
+              "type": "string",
+              "description": "Dex signing key expiration"
```

Having in mind that this will end up in the docs and in the editor's autocomplete suggestions let's try to be more specific

```suggestion
              "description": "Dex signing key expiration time duration (default 6h)."
```